keras/backend/tensorflow_backend.py

```diff
@@ -2629,7 +2629,9 @@ def slice(x, start, size):
 
     {{np_implementation}}
     """
-    if not (len(int_shape(x)) == len(start) == len(size)):
+    len_start = int_shape(start)[0] if is_tensor(start) else len(start)
```

Comment:
Please skip the check if `int_shape` is None or `int_shape[0]` is None (same below).

Reply:
@fchollet, Updated. Did I understand correctly?